rabbit-escape-engine/src/rabbitescape/engine/util/LookupTable2D.java

```diff
@@ -149,4 +149,9 @@ private int i( int c )
     {
         return list;
     }
+
+    public int itemCount()
+    {
+        return list.size();
+    }
 }
```

Comment:
I like size because it is short. I like itemCount as it is  a better description. size could be the 2D dimension. If you prefer size, we can go with that.

done.